src/shared/contexts/app.tsx

```diff
@@ -27,6 +28,17 @@ import {presentationMode as presentationModeCopy} from 'src/shared/copy/notifica
 import {AppState, TimeZone, Theme, NavBarState, FlowsCTA} from 'src/types'
 import {event} from 'src/cloud/utils/reporting'
 
+// Worker -- load prior to page load event, in order to intercept fetch in http/2.
+// see service worker life cycle. https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API/Using_Service_Workers
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import downloadWorker from 'worker-plugin/loader!../workers/downloadHelper'
```

Comment:
Just curious, why use `worker-plugin/loader!...` instead of the absolute file path `src/workers/downloadHelper`?

Reply:
This is required by the webpack build (note: this is different from jest) in order to load the service worker using the worker-plugin loader (under webpack 4).